### PR TITLE
Add applepay config on Magento backend

### PR DESCRIPTION
**Pull request by BushraAsif**

### CODE REVIEW

**Review by shahbaz-mehar — COMMENTED** [review]
Changes have been reviewed.

File: .github/workflows/code-analysis.yml [hunk 1]
```diff
@@ -3,9 +3,9 @@ name: PHP Code Analysis
 
 on:
   pull_request:
-    branches: [ main ]
+    branches: [ applepay-feature ]
```
Comment (emicha) [inline .github/workflows/code-analysis.yml:6]: You do not want to merge this to master.

Comment (BushraAsif) [reply .github/workflows/code-analysis.yml:6]: We want to merge it to the applepay-feature branch for now. When the functionality is complete we will merge the applepay-feature branch to the main branch.

Comment (BushraAsif) [reply .github/workflows/code-analysis.yml:6]: And we will revert this change when we create a pull request for the main branch.

**Review by emicha — APPROVED** [review]

**Pull request merged**